app-editors/kakoune/kakoune-2020.09.01.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8 should work here.

Reply:
Done. I haven't reviewed the details of EAPI 8 so I avoided using it, thanks.